app/services/funnel/registration/add_mfa.rb

```diff
@@ -4,22 +4,11 @@ class AddMfa
       def self.call(user_id, mfa_method, analytics)
         now = Time.zone.now
         funnel = RegistrationLog.find_by(user_id: user_id)
-        return if funnel.blank? || funnel.second_mfa.present?
+        funnel ||= RegistrationLog.create(user_id: user_id, submitted_at: now)
```

Comment:
Maybe we could save a step with:
```ruby
RegistrationLog.where(user_id: user_id).first_or_create(submitted_at: now)
```

I know that `create_or_find_by` is slightly safer, but it looks like with all the non-null columns in the DB that would cause problems

Reply:
yeah tried it and unfortunately submitted_at is non null.  that will be culled in a subsequent pr.